docs-e2e-testing/e2e/jest-preview.cy.js

```diff
@@ -1,7 +1,41 @@
 describe('Tests for Jest Preview', () => {
-  it('Check Jest Preview website is alive', () => {
+
+  it('Check internal pages are present, links and buttons work as expected', () => {
     cy.visit('https://www.jest-preview.com/')
     cy.contains('h1', 'Jest Preview')
-    cy.contains('Debug your Jest tests. Effortlessly.')
+    
+    // Check CTA Get Started should directed users to Docs page
+      cy.get('.button').click()
+      cy.url()
+        .should('contain', 'docs/getting-started/intro')
+      cy.contains('h1','Introduction')
+
+    // Check Docs page is present
+    cy.contains("Docs").click()
+      cy.url()
+        .should('contain', 'docs/getting-started/intro')
+      cy.contains('h1','Introduction')
+    
+    // Check Blog page is present
+    cy.contains("Blog").click()
+    cy.url()
+      .should('contain', '/blog')
+    cy.contains('Recent posts')
+    
+    // Check API page is present
+    cy.contains("API").click()
+    cy.url()
+      .should('contain', '/api')
+    cy.contains('Getting Started')
+
+    // Check Demo link is correct and Demo page can be loaded
+    cy.contains('Demo').should('have.attr', 'href', 'https://stackblitz.com/edit/jest-preview?file=README.md')
+    cy.contains('Demo').then(($a) => {
+      // pull off the fully qualified href from the <a>
+      const url = $a.prop('href')
+    
+      // make a cy.request to it
+      cy.request(url).its('body').should('include', 'Jest Preview')
```

Comment:
@huyenuet Actually, we don't need to verify `stackblitz.com`. Asserting the URL is enough in this case.